ts/utils/wallet.ts

```diff
@@ -13,19 +12,11 @@ import { NumberFromString } from "./number";
 export const FOUR_UNICODE_CIRCLES = "●".repeat(4);
 
 export const isExpiredCard = (creditCard: CreditCard) => {
-  const decodedValueYear = NumberFromString.decode(creditCard.expireYear);
-  const ccExpireYear = decodedValueYear.isRight()
-    ? decodedValueYear.value
-    : undefined;
-  const decodedValueMonth = NumberFromString.decode(creditCard.expireMonth);
-  const ccExpireMonth = decodedValueMonth.isRight()
-    ? decodedValueMonth.value
-    : undefined;
-  // if we can't decode month or year value, card will be considered as expired
-  if (ccExpireYear === undefined || ccExpireMonth === undefined) {
-    return true;
-  }
-  return isExpired(ccExpireMonth, ccExpireYear);
+  const { expireYear, expireMonth } = creditCard;
+
+  // In case the date is not valid, this call will return -1
+  // thus considering the card as expired
+  return isExpired(Number(expireMonth), Number(expireYear));
```

Comment:
this function changes the previous behavior:

> // if we can't decode month or year value, card will be considered as expired

If you run this test

`expect(isExpired(Number("AAA"), Number("BBB"))).toBe(true);`

it fails